testbed/stack.ts

```diff
@@ -30,6 +30,15 @@ export class Testbed extends cdk.Stack {
                 },
             ],
         });
+
+        //Tag pub subnets for KIT CP
+        const selection = vpc.selectSubnets({
+            subnetType: ec2.SubnetType.PUBLIC
+        });
```

Comment:
Why do we have public subnet for control plane nodes?

Reply:
It's not mandatory. As per our discussion, KIT has requirement of having non-overlapping subnet for DP and CP, so it's cleaner this way to pick public subnets for CP and private subnets for DP.

Reply:
We can also create separate private subnets for CP and DP for a cleaner separation?